k8s/Makefile

```diff
@@ -13,7 +13,7 @@ RCLONE_IMG ?= ${DOCKERHUB_USERNAME}/seldon-rclone:${CUSTOM_IMAGE_TAG}
 SCHEDULER_IMG ?= ${DOCKERHUB_USERNAME}/seldon-scheduler:${CUSTOM_IMAGE_TAG}
 
 MLSERVER_IMG ?= seldonio/mlserver:1.2.1
-TRITON_IMG ?= nvcr.io/nvidia/tritonserver:22.05-py3
+TRITON_IMG ?= nvcr.io/nvidia/tritonserver:22.11-py3
```

Comment:
any particular reason why are upgrading?

Reply:
the pytorch backend is main reason as its tied to a particular version of pytorch